eth/tracers/tracer.go

```diff
@@ -561,6 +568,25 @@ func wrapError(context string, err error) error {
 	return fmt.Errorf("%v    in server-side tracer function '%v'", err, context)
 }
 
+func (jst *Tracer) CapturePreEVM(env *vm.EVM, inputs map[string]interface{}) error {
```

Comment:
Other tracer `Capture` methods are defined as interface methods in core/vm/logger.go's `Tracer` type. This interface is implemented by `JSONLogger`, `StructLogger`, and `Tracer`. Should `CapturePreEVM` be likewise defined in that interface? Why or why not?

Reply:
Initially, I left it out of the interface as it will only be used for the JS tracers and not the rest.
On the other hand is a good addition, and later on rest tracers might have some benefits using it.
So bbc4a7b adds `CapturePreEVM` in the interface, to make everything cleaner.

Thanks @meowsbits.